frontstage/views/secure_messaging/create_message.py

```diff
@@ -73,14 +62,22 @@ def send_message(party_id, is_draft, case_id, survey, ru_ref):
     # If message has previously been saved as a draft add through the message id
     if form["msg_id"].data:
         message_json["msg_id"] = form['msg_id'].data
-    response = api_call('POST', endpoint, parameters={"is_draft": is_draft},
+    # Without is_draft parameter, date/time on the message doesn't get saved correctly,
+    # resulting in missing date/time in conversation list.
```

Comment:
Probably yea.   I'm not proud of this bit, but because we're going to be reworking secure-message soon, I don't want to dunk a lot of time into figuring out how 'is_draft' makes the time not appear when you get the thread list down the line.
The comment at least gives some context as to why that seems to 'randomly' be there and will hopefully be fixed down the line.